tests/RobotFramework/tests/cumulocity/configuration/configuration_operation.robot

```diff
@@ -108,6 +108,22 @@ Manual config_snapshot operation request
     ...    expected_status=successful
     ...    c8y_fragment=c8y_UploadConfigFile
 
+Config_snapshot operation request with the tedgeUrl created by agent
+    Set Device Context    ${PARENT_SN}
+    ${timestamp}=    Get Unix Timestamp
+    Publish and Verify Local Command
+    ...    topic=te/device/main///cmd/config_snapshot/tedge-configuration-plugin-1111
```

Comment:
For consistency it would be better to use the same command id as used in other tests, e.g. `local-3333` (though the id should still be unique to avoid any unexpected side-effects from other tests)


```suggestion
    ...    topic=te/device/main///cmd/config_snapshot/local-3333
```

Reply:
Included in 561e74603322e6811a45cac6c3cc50ab47eba56f